torch/_dynamo/config.py

```diff
@@ -66,6 +66,11 @@
 # any guards due to wobbles in shape, we recompile with *all* the wobbled shapes as being marked dynamic.
 automatic_dynamic_shapes = True
 
+# This flag changes how the shapes of parameters are treated. 
+# If this flag is set to False, then the shapes of torch.nn.Parameter are assumed to be static.
+# If this flag is set to True, then the shapes of torch.nn.Parameter are assumed to be dynamic
+assume_parameters_shapes_static_by_default = False
```

Comment:
I think you inverted the naming, False would imply that you are NOT assuming the parameters static by default.